src/App.tsx

```diff
@@ -33,171 +77,124 @@ export const App: React.FC = () => {
-              data-cy="FilterLinkAll"
-            >
-              All
-            </a>
-
-            <a
-              href="#/active"
-              className="filter__link"
-              data-cy="FilterLinkActive"
-            >
-              Active
-            </a>
 
-            <a
-              href="#/completed"
-              className="filter__link"
-              data-cy="FilterLinkCompleted"
+            {/* Active filter should have a 'selected' class */}
+            <nav className="filter" data-cy="Filter">
+              <a
+                href="#/"
+                className={cn('filter__link',
+                  { selected: filter === Status.All })}
+                data-cy="FilterLinkAll"
+                onClick={() => handleFilter(Status.All)}
+              >
+                {Status.All}
+              </a>
+
+              <a
+                href="#/active"
+                className={cn('filter__link',
+                  { selected: filter === Status.Active })}
+                data-cy="FilterLinkActive"
+                onClick={() => handleFilter(Status.Active)}
+              >
+                {Status.Active}
+              </a>
+
+              <a
+                href="#/completed"
+                className={cn('filter__link',
+                  { selected: filter === Status.Completed })}
+                data-cy="FilterLinkCompleted"
+                onClick={() => handleFilter(Status.Completed)}
+              >
+                {Status.Completed}
+              </a>
+            </nav>
+
+            {/* don't show this button if there are no completed todos */}
+            <button
+              type="button"
+              className="todoapp__clear-completed"
+              data-cy="ClearCompletedButton"
             >
-              Completed
-            </a>
-          </nav>
+              Clear completed
+            </button>
+          </footer>
+        )}
 
-          {/* don't show this button if there are no completed todos */}
-          <button
-            type="button"
-            className="todoapp__clear-completed"
-            data-cy="ClearCompletedButton"
-          >
-            Clear completed
-          </button>
-        </footer>
       </div>
 
       {/* Notification is shown in case of any error */}
       {/* Add the 'hidden' class to hide the message smoothly */}
       <div
         data-cy="ErrorNotification"
-        className="notification is-danger is-light has-text-weight-normal"
+        className={cn(
+          'notification is-danger is-light has-text-weight-normal',
+          { hidden: !errorMessage },
+        )}
       >
-        <button data-cy="HideErrorButton" type="button" className="delete" />
+        <button
+          data-cy="HideErrorButton"
+          type="button"
+          className="delete"
+          onClick={handleCloseError}
+        />
         {/* show only one message at a time */}
         Unable to load todos
```

Comment:
Try to show `{errorMessage}` there, in next step of this task.

Reply:
agree with `Xilston` student